<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snowgui::CatalogDialog</class>
 <widget class="QDialog" name="snowgui::CatalogDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="catalogBox"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="objectnameField"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="objectLabel">
     <property name="text">
      <string>Object:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="searchButton">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="objectnameLabel">
     <property name="text">
      <string>Object Name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="catalogLabel">
     <property name="text">
      <string>Catalog:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLabel" name="objectField">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
